### DISCUSSION TITLE
Stuck during plot？

### DISCUSSION THREAD

**Discussion by TianciD** [Chia-Network/chia-blockchain#12179] [Support]
Compressing tables 6 and 7
Progress update: 0.980
	Bucket 0 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 1 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 2 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 3 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 4 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 5 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 6 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 7 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 8 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 9 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 10 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 11 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 12 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 13 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 14 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 15 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 16 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 17 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 18 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 19 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 20 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 21 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 22 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 23 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 24 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 25 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 26 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 27 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 28 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 29 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 30 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 31 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 32 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 33 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 34 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 35 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 36 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 37 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 38 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 39 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 40 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 41 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 42 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 43 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 44 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 45 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 46 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 47 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 48 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 49 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 50 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 51 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 52 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 53 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 54 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 55 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 56 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 57 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 58 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 59 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 60 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 61 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 62 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 63 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 64 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 65 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 66 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 67 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 68 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 69 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 70 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 71 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 72 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 73 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 74 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 75 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 76 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 77 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 78 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 79 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 80 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 81 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 82 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 83 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 84 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 85 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 86 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 87 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 88 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 89 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 90 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 91 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 92 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 93 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 94 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 95 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 96 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 97 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 98 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 99 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 100 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 101 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 102 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 103 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 104 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 105 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 106 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 107 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 108 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 109 uniform sort. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.250GiB.
	Bucket 110 QS. Ram: 3.250GiB, u_sort min: 0.500GiB, qs min: 0.171GiB. force_qs: 1
	First computation pass time: 1710.616 seconds. CPU (32.910%) Fri Jul  1 12:45:25 2022
	Bucket 0 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.501GiB.
	Bucket 1 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 2 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 3 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 4 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 5 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 6 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 7 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 8 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 9 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 10 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.501GiB.
	Bucket 11 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 12 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 13 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 14 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 15 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 16 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 17 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 18 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 19 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 20 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 21 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 22 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 23 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 24 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.501GiB.
	Bucket 25 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 26 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 27 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 28 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 29 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 30 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 31 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 32 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 33 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 34 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.501GiB.
	Bucket 35 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 36 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 37 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 38 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 39 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 40 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 41 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 42 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 43 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 44 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 45 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 46 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 47 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 48 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 49 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 50 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 51 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.501GiB.
	Bucket 52 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 53 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 54 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 55 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 56 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 57 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 58 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 59 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 60 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 61 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.501GiB.
	Bucket 62 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 63 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 64 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.501GiB.
	Bucket 65 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 66 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.501GiB.
	Bucket 67 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 68 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 69 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.501GiB.
	Bucket 70 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 71 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 72 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.501GiB.
	Bucket 73 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 74 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 75 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 76 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 77 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.501GiB.
	Bucket 78 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 79 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 80 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 81 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 82 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 83 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 84 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.501GiB.
	Bucket 85 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 86 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 87 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 88 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 89 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 90 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 91 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 92 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.501GiB.
	Bucket 93 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 94 uniform sort. Ram: 3.250GiB, u_sort min: 1.500GiB, qs min: 0.502GiB.
	Bucket 95 QS. Ram: 3.250GiB, u_sort min: 0.750GiB, qs min: 0.357GiB. force_qs: 1
	Second computation pass time: 1419.223 seconds. CPU (41.060%) Fri Jul  1 13:09:12 2022
	Wrote 4295429710 entries
Total compress table time: 3137.025 seconds. CPU (36.530%) Fri Jul  1 13:09:12 2022
Time for phase 3 = 17367.886 seconds. CPU (45.460%) Fri Jul  1 13:09:16 2022

Starting phase 4/4: Write Checkpoint tables into "/Volumes/disk1/chiaplot_temp/plot-k32-2022-07-01-00-08-1af2q6a69969aabdd649f0c97924e2f14f123f45aaa1123fhyx466q3ac45.plot.2.tmp" ... Fri Jul  1 13:09:16 2022
	Starting to write C1 and C3 tables
	Bucket 0 uniform sort. Ram: 3.250GiB, u_sort min: 1.125GiB, qs min: 0.281GiB.

**Stuck at this step, I see that the process is no longer consuming resources, although it still exists.**

**Comment by TianciD** [comment]
It happened again today, same with yesterday.

**Reply by loppefaaret** [reply]
is this on macOS ? what version is it ? and what plotter are you using ? you dont see any diskactivity ?

**Reply by TianciD** [reply]
Yes it's MacOs and the plotter I use is Chia Proof of Space 1.0.9, hard drive is Seagate 16T

It happened again today, at first I thought it was me running multiple tasks on one HDD at the same time, then I kept only one task and the problem happened again. Now I suspect a hard drive cooling issue and I'm trying to improve the cooling environment.

Problems continue to be explored...

**Comment by TianciD** [comment]
It happened again today, but after a while, it got better again, but I'm sure the chia process has no cpu usage and no disk activity during the stop

**Reply by TianciD** [reply]
I only keep a chia process that I'm encapsulating with an external hard drive enclosure connected via usb3.0
